cadence/tests/flow_evm_bridge_tests.cdc

```diff
@@ -701,6 +709,40 @@ fun testBatchOnboardByEVMAddressSucceeds() {
 
 /* --- BRIDGING NFTS - Test bridging both Cadence- & EVM-native NFTs --- */
 
+access(all)
+fun testPauseBridgeSucceeds() {
+    // Pause the bridge
+    let pauseResult = executeTransaction(
+        "../transactions/bridge/admin/pause/pause_bridge.cdc",
+        [],
+        bridgeAccount
+    )
+    Test.expect(pauseResult, Test.beSucceeded())
+
+    var aliceOwnedIDs = getIDs(ownerAddr: alice.address, storagePathIdentifier: "cadenceExampleNFTCollection")
+    Test.assertEqual(1, aliceOwnedIDs.length)
+
+    var aliceCOAAddressHex = getCOAAddressHex(atFlowAddress: alice.address)
+
+    // Execute bridge to EVM - should fail after pausing
+    bridgeNFTToEVM(
+        signer: alice,
+        contractAddr: exampleNFTAccount.address,
+        contractName: "ExampleNFT",
+        nftID: aliceOwnedIDs[0],
+        bridgeAccountAddr: bridgeAccount.address,
+        beFailed: true
+    )
+
+    // Unpause bridging
+    let unpauseResult = executeTransaction(
+        "../transactions/bridge/admin/pause/unpause_bridge.cdc",
+        [],
+        bridgeAccount
+    )
+    Test.expect(unpauseResult, Test.beSucceeded())
+}
```

Comment:
Maybe should have a check here that the bridge is unpaused?

Reply:
Done in [36718c2](https://github.com/onflow/flow-evm-bridge/pull/52/commits/36718c2aec09d8c080279f6bb799885a142c3dfc)